client/my-sites/stats/stats-page-placeholder/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+const StatsPagePlaceholder = () => {
```

Comment:
Don't need to change, but a pattern to consider for fully static components:

https://babeljs.io/docs/plugins/transform-react-constant-elements/

i.e. moving into a top-level constant will allow React to skip reconciling on rerenders.